coordinator/app/src/main/kotlin/net/consensys/zkevm/coordinator/blockcreation/BlockCreationMonitor.kt

```diff
@@ -21,7 +24,7 @@ import kotlin.time.Duration.Companion.days
 
 class BlockCreationMonitor(
   private val vertx: Vertx,
-  private val extendedWeb3j: ExtendedWeb3J,
+  private val web3j: Web3j,
```

Comment:
Why are we going back to plain Web3j here and in other places?

Reply:
Because it was simpler this way and this extension is not properly designed and refactoring adapting would be more work. We created this interface to basically facilitate mock<> and return responses, if we create a fake rpc client this wont's be necessary, we can tests against state that concrete implementations. That's my pan for new tests of BlockCreationMonitor.

Reply:
`getNetNextSafeBlock` is more complex now. Is it possible to make this change later, at the same time with the new tests?

Reply:
done